src/api/app/models/project.rb

```diff
@@ -145,14 +145,14 @@ def self.restore(project_name, backend_opts = {})
     project = Project.new(name: project_name)
 
     Project.transaction do
-      project.update_from_xml!(Xmlhash.parse(project.meta.to_s))
+      project.update_from_xml!(Xmlhash.parse(project.meta.content))
```

Comment:
is there any value in having meta not returning content right away? (just curious as you it .meta.content appears quite often in your PR)

Reply:
We would have to look closer into this. But you are right it seems that we only use the content right now.
The other thing is that there is a similar method called config and might be some others. I would keep them in sync.